v0/destinations/klaviyo/transform.js

```diff
@@ -160,13 +163,75 @@ const trackRequestHandler = (message, category, destination) => {
     WhiteListedTraits
   );
   customerProperties = removeUndefinedAndNullValues(customerProperties);
-  if (destination.Config.enforceEmailAsPrimary) {
-    delete customerProperties.$id;
-    customerProperties._id = getFieldValueFromMessage(message, "userId");
-  }
-  payload.customer_properties = customerProperties;
-  if (message.timestamp) {
-    payload.time = toUnixTimestamp(message.timestamp);
+  return customerProperties;
+};
+const trackRequestHandler = (message, category, destination) => {
+  let payload = {};
+  let event = getValueFromMessage(message, "event");
```

Comment:
Let's use simple get() call to extract `event`

Reply:
Everywhere in transform.js getValueFromMessage is being used , so let's maintain parity if there is no advantage in changing that.

Reply:
Actually we are using only `get()` and `getFieldValueFromMessage()` in transform.js so directly calling `get()` here would be better.

Reply:
If we need to extract value from a key that does not have multiple sources (say `userId` --- `userId`, `traits.userId`, `traits.id` etc.), then `get` is ideal, as in this case. Otherwise, `getFieldValueFromMessage` should be used.